webservices/legal_docs/load_legal_docs.py

```diff
@@ -448,7 +448,7 @@ def load_archived_murs(from_mur_no=None, specific_mur_no=None, num_processes=1,
     """
     Reads data for archived MURs from http://classic.fec.gov/MUR, assembles a JSON
     document corresponding to the MUR and indexes this document in Elasticsearch
```

Comment:
line 449:  should be http://classic.fec.gov/MUR/  (with end `/`), otherwise it is a broken link. Thanks